parsl/executors/high_throughput/interchange.py

```diff
@@ -621,7 +621,7 @@ def start_file_logger(filename: str, level: int = logging.DEBUG, format_string:
         None.
     """
     if format_string is None:
-        format_string = "%(asctime)s.%(msecs)03d %(name)s:%(lineno)d %(processName)s(%(process)d) %(threadName)s %(funcName)s [%(levelname)s]  %(message)s"
+        format_string = "%(asctime)s.%(msecs)03d %(name)s:%(lineno)d %(processName)s %(threadName)s %(funcName)s [%(levelname)s]  %(message)s"
```

Comment:
it's not so good to remove information from the logging messages - in the past we've been making these log lines *longer* not shorter, to get more information - especially to do with processes and threads because that can be hard to debug.

probably a better thing to do here would have been to split the format string over multiple lines - have a look at how PR #3130 addressed this line being too long, for example